<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>


    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet"
        integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous">
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Oswald:wght@200&display=swap" rel="stylesheet">
    <link href=".\styles.css" rel="stylesheet">


    <style>
        .btn-custom {
            color: #fff;
            background-color: #d67a41;
            border-color: #f6a776;
        }

        .link {
            color: #586d54;
            text-decoration: none;
        }

        .link:hover {
            color: #364433;
        }
    </style>
    <title>Research</title>
</head>



<body>
    <div class="background-image" style="background-image: url('./Media/navbar_2.jpg');"></div>
    <!-- Navbar -->
    <div id="navbarJQ"></div>
    <script>
        $("#navbarJQ").load("navbar.html");
    </script>




    <div style="padding-top: 20vh;"></div>

    <!-- <section class="container-fluid pt-5 content-publications">
        <div class="row">
            <div class="col-6">
                Google Scholar
            </div>
            <div class="col-6">

            </div>
        </div>
    </section> -->

    <!-- PUBLICATION SECTION 1-->
    <section class="container-fluid pt-5 content-publications">
        <div class="row p-3">
            <div class="col-12 hidden align-items-center">
                <hr>
                <a class="link" href="https://doi.org/10.1111/jbi.15022"><b>Mikkelsen, E. K.</b>, Lavareda, D., Vallinoto, M., Aleixo, A. & Weir, J. 2024.
                    Phylogeography of the Red‐Headed Manakin Supports the River‐Refuge Hypothesis. Journal of Biogeography 52(1), 92-107.</a>
            </div>
            <div class="col-12">
                <!-- COLLAPSED SECTION -->
                <div class="container mt-1">
                    <div class="row ">
                        <!-- Expeditions and game collapsed content -->
                        <div class="col">
                            <div class="d-flex justify-content-end mb-3">
                                <button class="btn btn-custom" type="button" data-toggle="collapse"
                                    data-target="#PUBSECT_1" aria-expanded="false" aria-controls="PUBSECT_1">
                                    Abstract
                                </button>
                            </div>
                            <div class="collapse collapse-content" id="PUBSECT_1">
                                <div class="card card-body">
                                    <!-- CONTENT of collapsed section goes here -->
                                    <div class="row p-3 mb-5 align-items-center">
                                        <div class="col">
                                            <img src="https://github.com/elsemikk/Ceratopipra_rubrocapilla_Phylogeography/raw/master/1_Datasets/Header_image.png" class="img-fluid" alt="galleryPhoto">
                                            <b>Aim</b>
                                            The Amazon rainforest is one of the most biodiverse regions on earth, but our understanding of the processes that have shaped its patterns of diversity remains incomplete. One hypothesis for Amazonian speciation is the river-refuge hypothesis, which suggests that retraction of forests away from the periphery of Amazonia, where rivers are narrow, facilitated divergence of populations separated by wide rivers. Later re-expansion of forests would have allowed secondary contact between these populations, and co-occurring hybrid zones may reveal the location where expanding forests first reconnected. Here, we test whether a widespread Amazonian songbird species shows evidence of population contact zones in the eastern headwaters of the Tapajós river, hypothesised to be an area of secondary contact under the river-refuge hypothesis.
                                            
                                            <b>Location</b>
                                            Amazon and Atlantic forests of South America.
                                            
                                            <b>Taxon</b>
                                            <em>Ceratopipra rubrocapilla</em> (Pipridae, Passeriformes).
                                            
                                            <b>Methods</b>
                                            We sampled 147 <em>C. rubrocapilla</em> (Red-headed Manakin) across its vast range, with 70 samples sequenced using reduced-representation genomic sequencing and 139 sequenced at the mitochondrial gene cytochrome b. We use population genetic and phylogenetic analyses to investigate patterns of gene flow and population structure across the range, with a particular focus on samples from the headwaters of the Tapajós River to evaluate the potential role of the river-refuge hypothesis.

                                            <b>Results</b>
                                            We observe shallow population structure and evidence for widespread gene flow across the range of <em>C. rubrocapilla</em>. Patterns in the Tapajós headwaters match predictions of the river-refuge hypothesis, with eastern headwater samples most similar to samples farther west on the opposite bank of the Tapajós River.
                                            
                                            <b>Main Conclusions</b>
                                            The close co-occurrence between population contact zones within <em>C. rubrocapilla</em> and other previously studied hybrid zones reinforces the hypothesis that the eastern Tapajós headwaters is where rainforest populations reconnected in the past, following predictions of the river-refuge hypothesis of Amazonian speciation.
                                            

                                        </div>
                                    </div>

                                </div>

                            </div>
                        </div>
                    </div>
                </div>
            </div>
            <hr>
        </div>




    </section>

    <section class="container-fluid pt-5 content-publications">
        <div class="row p-3">
            <div class="col-12 hidden align-items-center">
                <hr>
                <a class="link" href="https://doi.org/10.1093/sysbio/syac078"><b>Mikkelsen, E. K.</b> & Weir, J. 2023.
                    Phylogenomics Reveals that Mitochondrial Introgression with Limited Nuclear Introgression
                    Characterizes
                    Skua Species Proposed to be of Hybrid Origin. Systematic Biology. 72(1):78-91.</a>
            </div>
            <div class="col-12">
                <!-- COLLAPSED SECTION -->
                <div class="container mt-1">
                    <div class="row ">
                        <!-- Expeditions and game collapsed content -->
                        <div class="col">
                            <div class="d-flex justify-content-end mb-3">
                                <button class="btn btn-custom" type="button" data-toggle="collapse"
                                    data-target="#PUBSECT_1" aria-expanded="false" aria-controls="PUBSECT_1">
                                    Abstract
                                </button>
                            </div>
                            <div class="collapse collapse-content" id="PUBSECT_1">
                                <div class="card card-body">
                                    <!-- CONTENT of collapsed section goes here -->
                                    <div class="row p-3 mb-5 align-items-center">
                                        <div class="col">
                                            The skuas and jaegers (Stercorariidae) are an enigmatic family of seven seabird species that breed at Arctic and Antarctic latitudes. 
                                            The phylogenetic relationships amongst the species have been controversial, with one of the biggest enigmas involving the Pomarine Jaeger (<em>Stercorarius pomarinus</em>), which has been proposed to represent a hybrid species originating from the merging of distant lineages within the complex. 
                                            We inferred a phylogeny for the family using multispecies coalescent methods with whole-genome sequencing for all seven species of Stercorariidae, and document an evolutionary history rich in introgression. 
                                            We uncover evidence for mitochondrial capture and nuclear introgression between <em>S. pomarinus</em> and <em>Stercorarius skua</em>, providing a potential avenue for adaptive introgression. 
                                            One candidate for adaptive introgression is the MC1R plumage gene which appears to have introgressed from one of the large skuas into <em>S. pomarinus</em>, where it now forms the basis of the dark-morph color polymorphism of that species. 
                                            We further highlight a complex biogeographical history of interchange between the Arctic and Antarctic, with unexpected close ancestry between <em>S. skua</em> of the northern hemisphere and <em>Stercorarius antarcticus</em> of the southern hemisphere. 
                                            These results highlight the dynamic history of introgression during pelagic seabird radiation.
                                            <img src="https://github.com/elsemikk/website/blob/test_GCU/Media/Stercorariidae_graphical_abstract.png?raw=true" class="img-fluid" alt="galleryPhoto">
                                        </div>
                                    </div>

                                </div>

                            </div>
                        </div>
                    </div>
                </div>
            </div>
            <hr>
        </div>




    </section>

    <!-- PUBLICATION SECTION 2-->
    <section class="container-fluid content-publications">
        <div class="row p-3">
            <div class="col-12 hidden align-items-center">
                <a class="link" href="https://doi.org/10.1093/pnasnexus/pgac074">de Zwaan, D., Mackenzie, J.,
                    <b>Mikkelsen,E.K.</b> , Wood, C., & Wang, S. 2022. Pleiotropic opposing dominance within a color
                    gene block contributes to a nascent species boundary via its influence on hybrid male territorial
                    behavior. PNAS Nexus. 1(3):pgac074.</a>
            </div>
            <div class="col-12">
                <!-- COLLAPSED SECTION -->
                <div class="container mt-1">
                    <div class="row ">
                        <!-- Expeditions and game collapsed content -->
                        <div class="col">
                            <div class="d-flex justify-content-end mb-3">
                                <button class="btn btn-custom" type="button" data-toggle="collapse"
                                    data-target="#PUBSECT_2" aria-expanded="false" aria-controls="PUBSECT_2">
                                    Abstract
                                </button>
                            </div>
                            <div class="collapse collapse-content" id="PUBSECT_2">
                                <div class="card card-body">
                                    <!-- CONTENT of collapsed section goes here -->
                                    <div class="row p-3 mb-5 align-items-center">
                                        <div class="col">
                                            The divergence of plumage color genes contributes to songbird radiation. 
                                            However, the mechanisms by which color gene divergence counteracts gene flow to maintain reproductive isolation during the formation of new species boundaries remain elusive. 
                                            The hybrid zone between <em>Setophaga occidentalis</em> (SOCC) and <em>S. townsendi</em> (STOW) in the Cascade Range provides a natural observatory to investigate potential behavioral mechanisms underlying divergent selection on color genes. 
                                            Recently, we found that selection within a single gene block associated with plumage color variation has maintained a stable and narrow hybrid zone. 
                                            Here, we investigated the potential role of plumage signals in moderating a behavioral mechanism of selection. 
                                            Specifically, we assessed whether two plumage traits are associated with body size among breeding males and if trait mismatch predicted aggressive behavior within hybrid and parental individuals in response to simulated territorial intrusion. 
                                            The two plumage signals, cheek and flank coloration, though associated with the same gene block, reflect opposing dominance of SOCC and STOW alleles. We found that both plumage traits significantly predict the body size in the territorial sex (i.e. males). 
                                            The opposing dominance of the single color gene block resulted in plumage signal discordance in heterozygotes, which in turn was associated with reduced hybrid territorial performance, an important proxy of fitness in this system. 
                                            Taken together, these observations point to a single-locus-two-alleles mechanism of incompatibility in shaping a natural species boundary in the early stage of speciation.
                                        </div>
                                    </div>

                                </div>

                            </div>
                        </div>
                    </div>
                </div>
            </div>
            <hr>
        </div>




    </section>

    <!-- PUBLICATION SECTION 3-->
    <section class="container-fluid content-publications">
        <div class="row p-3">
            <div class="col-12 hidden align-items-center">
                <a class="link" href="https://doi.org/10.1111/mec.16015"><b>Mikkelsen, E. K.</b> & Irwin, D. 2021.
                    Ongoing production of low-fitness hybrids limits range overlap between divergent cryptic species.
                    Molecular Ecology. 30:4090-4102.</a>
            </div>
            <div class="col-12">
                <!-- COLLAPSED SECTION -->
                <div class="container mt-1">
                    <div class="row ">
                        <!-- Expeditions and game collapsed content -->
                        <div class="col">
                            <div class="d-flex justify-content-end mb-3">
                                <button class="btn btn-custom" type="button" data-toggle="collapse"
                                    data-target="#PUBSECT_3" aria-expanded="false" aria-controls="PUBSECT_3">
                                    Abstract
                                </button>
                            </div>
                            <div class="collapse collapse-content" id="PUBSECT_3">
                                <div class="card card-body">
                                    <!-- CONTENT of collapsed section goes here -->
                                    <div class="row p-3 mb-5 align-items-center">
                                        <div class="col">
                                            Contact zones between recently diverged taxa provide opportunities to examine the causes of reproductive isolation and the processes that determine whether two species can coexist over a broad region. The Pacific wren (<em>Troglodytes pacificus</em>) and winter wren (<em>Troglodytes hiemalis</em>) are two morphologically similar songbirds that started diverging about 4 million years ago, older than most sister species pairs of temperate songbirds. The ranges of these species come into narrow contact in western Canada, where the two species remain distinct. To assess evidence for differentiation, hybridization and introgression in this system, we examined variation in over 250,000 single nucleotide polymorphism markers distributed across the genome. The two species formed highly divergent genetic clusters, consistent with long-term differentiation. In a set of 75 individuals, two first-generation hybrids (i.e., F1’s) were detected, indicating only moderate levels of assortative mating between these taxa. We found no recent backcrosses or other evidence of recent breeding success of F1’s, indicating very low or zero fitness of F1 hybrids. Examination of genomic variation shows evidence for only a single backcrossing event many generations ago. The moderate rate of hybridization combined with very low F1 hybrid fitness is expected to result in a population sink in the contact zone, largely explaining the narrow overlap of the two species. If such dynamics are common in nature, they could explain the narrow range overlap often observed between pairs of closely related species.

                                        </div>
                                    </div>

                                </div>

                            </div>
                        </div>
                    </div>
                </div>
            </div>
            <hr>
        </div>




    </section>

    <!-- PUBLICATION SECTION 4-->
    <section class="container-fluid content-publications">
        <div class="row p-3">
            <div class="col-12 hidden align-items-center">
                <a class="link" href="https://doi.org/10.1098/rspb.2021.2362">Bemmels, J. B., <b>Mikkelsen, E. K.</b>,
                    Haddrath, O., Colbourne, R. M., Robertson, H. A. & Weir, J. 2021. Demographic decline and
                    lineage-specific adaptations characterize New Zealand kiwi. Proceedings of the Royal Society B.
                    288(1965):20212362.</a>
            </div>
            <div class="col-12">
                <!-- COLLAPSED SECTION -->
                <div class="container mt-1">
                    <div class="row ">
                        <!-- Expeditions and game collapsed content -->
                        <div class="col">
                            <div class="d-flex justify-content-end mb-3">
                                <button class="btn btn-custom" type="button" data-toggle="collapse"
                                    data-target="#PUBSECT_4" aria-expanded="false" aria-controls="PUBSECT_4">
                                    Abstract
                                </button>
                            </div>
                            <div class="collapse collapse-content" id="PUBSECT_4">
                                <div class="card card-body">
                                    <!-- CONTENT of collapsed section goes here -->
                                    <div class="row p-3 mb-5 align-items-center">
                                        <div class="col">
                                            Small and fragmented populations may become rapidly differentiated due to genetic drift, making it difficult to distinguish whether neutral genetic structure is a signature of recent demographic events, or of long-term evolutionary processes that could have allowed populations to adaptively diverge. We sequenced 52 whole genomes to examine Holocene demographic history and patterns of adaptation in kiwi (<em>Apteryx</em>), and recovered 11 strongly differentiated genetic clusters corresponding to previously recognized lineages. Demographic models suggest that all 11 lineages experienced dramatic population crashes relative to early- or mid-Holocene levels. Small population size is associated with low genetic diversity and elevated genetic differentiation (<em>F</em>ST), suggesting that population declines have strengthened genetic structure and led to the loss of genetic diversity. However, population size is not correlated with inbreeding rates. Eight lineages show signatures of lineage-specific selective sweeps (284 sweeps total) that are unlikely to have been caused by demographic stochasticity. Overall, these results suggest that despite strong genetic drift associated with recent bottlenecks, most kiwi lineages possess unique adaptations and should be recognized as separate adaptive units in conservation contexts. Our work highlights how whole-genome datasets can address longstanding uncertainty about the evolutionary and conservation significance of small and fragmented populations of threatened species.
                                        </div>
                                    </div>

                                </div>

                            </div>
                        </div>
                    </div>
                </div>
            </div>
            <hr>
        </div>




    </section>

    <!-- PUBLICATION SECTION 5-->
    <section class="container-fluid content-publications">
        <div class="row p-3">
            <div class="col-12 hidden align-items-center">
                <a class="link" href="https://doi.org/10.1038/s41467-021-24586-8">Wang, S., Ore, M., <b>Mikkelsen, E.
                        K.</b> , Lee-Yaw, J., Toews, D., Rohwer, S., & Irwin, D. 2021. Signatures of mitonuclear
                    coevolution in a warbler species complex. Nature Communications. 12:4279.</a>
            </div>
            <div class="col-12">
                <!-- COLLAPSED SECTION -->
                <div class="container mt-1">
                    <div class="row ">
                        <!-- Expeditions and game collapsed content -->
                        <div class="col">
                            <div class="d-flex justify-content-end mb-3">
                                <button class="btn btn-custom" type="button" data-toggle="collapse"
                                    data-target="#PUBSECT_5" aria-expanded="false" aria-controls="PUBSECT_5">
                                    Abstract
                                </button>
                            </div>
                            <div class="collapse collapse-content" id="PUBSECT_5">
                                <div class="card card-body">
                                    <!-- CONTENT of collapsed section goes here -->
                                    <div class="row p-3 mb-5 align-items-center">
                                        <div class="col">
                                            Divergent mitonuclear coadaptation could facilitate speciation. We investigate this possibility in two hybridizing species of warblers, <em>Setophaga occidentalis</em> and <em>S. townsendi</em>, in western North America. Inland <em>S. townsendi</em> harbor distinct mitochondrial DNA haplotypes from those of <em>S. occidentalis</em>. These populations also differ in several nuclear DNA regions. Coastal <em>S. townsendi</em> demonstrate mixed mitonuclear ancestry from S. occidentalis and inland <em>S. townsendi</em>. Of the few highly-differentiated chromosomal regions between inland <em>S. townsendi</em> and <em>S. occidentalis</em>, a 1.2 Mb gene block on chromosome 5 is also differentiated between coastal and inland <em>S. townsendi</em>. Genes in this block are associated with fatty acid oxidation and energy-related signaling transduction, thus linked to mitochondrial functions. Genetic variation within this candidate gene block covaries with mitochondrial DNA and shows signatures of divergent selection. Spatial variation in mitonuclear ancestries is correlated with climatic conditions. Together, these observations suggest divergent mitonuclear coadaptation underpins cryptic differentiation in this species complex.

                                        </div>
                                    </div>

                                </div>

                            </div>
                        </div>
                    </div>
                </div>
            </div>
            <hr>
        </div>




    </section>

    <!-- PUBLICATION SECTION 6-->
    <section class="container-fluid content-publications">
        <div class="row p-3">
            <div class="col-12 hidden align-items-center">
                <a class="link" href="https://doi.org/10.1111/mec.16105">Bemmels, J., Bramwell, A., Anderson, S., Luzuriaga-Aveiga, V., <b>Mikkelsen, E. K.</b> , & Weir, J. 2021. Geographic contact drives genomic character displacement and increased reproductive isolation in two cryptic <em>Empidonax</em> flycatcher. Molecular Ecology. 30:4833–4844.</a>
            </div>
            <div class="col-12">
                <!-- COLLAPSED SECTION -->
                <div class="container mt-1">
                    <div class="row ">
                        <!-- Expeditions and game collapsed content -->
                        <div class="col">
                            <div class="d-flex justify-content-end mb-3">
                                <button class="btn btn-custom" type="button" data-toggle="collapse"
                                    data-target="#PUBSECT_6" aria-expanded="false" aria-controls="PUBSECT_6">
                                    Abstract
                                </button>
                            </div>
                            <div class="collapse collapse-content" id="PUBSECT_6">
                                <div class="card card-body">
                                    <!-- CONTENT of collapsed section goes here -->
                                    <div class="row p-3 mb-5 align-items-center">
                                        <div class="col">
                                            Geographic contact between sister lineages often occurs near the final stages of speciation, but its role in speciation's completion remains debated. 
Reproductive isolation may be essentially complete prior to secondary contact. 
Alternatively, costly interactions between partially reproductively isolated species – such as maladaptive hybridization or competition for resources – may select for divergence, increasing reproductive isolation and driving speciation toward completion. 
Here, we use coalescent demographic modelling and whole-genome data sets to show that a period of contact and elevated hybridization between sympatric eastern North American populations of two cryptic bird species preceded a major increase in reproductive isolation between these populations within the last 10,000 years. 
In contrast, substantial introgression continues to the present in a western contact zone where geographic overlap is much narrower and probably of more recent origin. 
In the sympatric eastern region where reproductive isolation has increased, it is not accompanied by character displacement in key morphometric traits, plumage coloration, or ecological traits. 
While the precise trait and underlying mechanism driving increased reproductive isolation remains unknown, we discuss several possibilities and outline avenues for future research. 
Overall, our results highlight how demographic models can reveal the geographic context in which reproductive isolation was completed, and demonstrate how contact can accelerate the final stages of speciation.
                                        </div>
                                    </div>

                                </div>

                            </div>
                        </div>
                    </div>
                </div>
            </div>
            <hr>
        </div>




    </section>

    <!-- PUBLICATION SECTION 7-->
    <section class="container-fluid content-publications">
        <div class="row p-3">
            <div class="col-12 hidden align-items-center">
                <a class="link" href="https://doi.org/10.1016/j.ygeno.2020.07.047"><b>Mikkelsen, E. K.</b>  & Weir, J. 2020. The genome of the Xingu scale-backed antbird (<em>Willisornis vidua nigrigula</em>) reveals lineage-specific adaptations. Genomics. 112(6):4552-4560.</a>
                    
            </div>
            <div class="col-12">
                <!-- COLLAPSED SECTION -->
                <div class="container mt-1">
                    <div class="row ">
                        <!-- Expeditions and game collapsed content -->
                        <div class="col">
                            <div class="d-flex justify-content-end mb-3">
                                <button class="btn btn-custom" type="button" data-toggle="collapse"
                                    data-target="#PUBSECT_7" aria-expanded="false" aria-controls="PUBSECT_7">
                                    Abstract
                                </button>
                            </div>
                            <div class="collapse collapse-content" id="PUBSECT_7">
                                <div class="card card-body">
                                    <!-- CONTENT of collapsed section goes here -->
                                    <div class="row p-3 mb-5 align-items-center">
                                        <div class="col">
                                            Antbirds (Thamnophilidae) are a large neotropical family of passerine bird renowned for the ant-following foraging strategies of several members of this clade. 
                                            The high diversity of antbirds provides ample opportunity for speciation studies, however these studies can be hindered by the lack of an annotated antbird reference genome. 
                                            In this study, we produced a high-quality annotated reference genome for the Xingu Scale-backed Antbird (<em>Willisornis vidua nigrigula</em>) using 10X Genomics Chromium linked-reads technology. 
                                            The assembly is 1.09 Gb, with a scaffold N50 of 12.1 Mb and 17,475 annotated protein coding genes. We compare the proteome of <em>W. v. nigrigula</em> to several other passerines, and produce annotations for two additional antbird genomes in order to identify genes under lineage-specific positive selection and gene families with evidence for significant expansions in antbirds. 
                                            Several of these genes have functions potentially related to the lineage-specific traits of antbirds, including adaptations for thermoregulation in a humid tropical environment.
                                        </div>
                                    </div>

                                </div>

                            </div>
                        </div>
                    </div>
                </div>
            </div>
            <hr>
        </div>




    </section>

    <!-- PUBLICATION SECTION 8-->
    <section class="container-fluid content-publications">
        <div class="row p-3">
            <div class="col-12 hidden align-items-center">
                <a class="link" href="https://doi.org/10.1111/evo.14103">Cronemberger, A. A., Aleixo, A., <b>Mikkelsen, E. K.</b> & Weir, J. T. 2020. Postzygotic isolation drives genomic speciation between highly cryptic <em>Hypocnemis</em>  antbirds from Amazonia. Evolution. 74:2512-2525.</a>
                    
            </div>
            <div class="col-12">
                <!-- COLLAPSED SECTION -->
                <div class="container mt-1">
                    <div class="row ">
                        <!-- Expeditions and game collapsed content -->
                        <div class="col">
                            <div class="d-flex justify-content-end mb-3">
                                <button class="btn btn-custom" type="button" data-toggle="collapse"
                                    data-target="#PUBSECT_8" aria-expanded="false" aria-controls="PUBSECT_8">
                                    Abstract
                                </button>
                            </div>
                            <div class="collapse collapse-content" id="PUBSECT_8">
                                <div class="card card-body">
                                    <!-- CONTENT of collapsed section goes here -->
                                    <div class="row p-3 mb-5 align-items-center">
                                        <div class="col">
                                            How species evolve reproductive isolation in the species-rich Amazon basin is poorly understood in vertebrates. 
Here, we sequenced a reference genome and used a genome-wide sample of SNPs to analyze a hybrid zone between two highly cryptic species of <em>Hypocnemis</em> warbling-antbirds—the Rondonia warbling-antbird (<em>H. ochrogyna</em>) and Spix's warbling-antbird (<em>H. striata</em>)—in a headwater region of southern Amazonia. 
We found that both species commonly hybridize, producing F1s and a variety of backcrosses with each species but we detected only one F2-like hybrid. 
Patterns of heterozygosity, hybrid index, and interchromosomal linkage disequilibrium in hybrid populations closely match expectations under strong postzygotic isolation. 
Hybrid zone width (15.4 km) was much narrower than expected (211 km) indicating strong selection against hybrids. 
A remarkably high degree of concordance in cline centers and widths across loci, and a lack of reduced interspecific <em>F</em>st between populations close to versus far from the contact zone, suggest that genetic incompatibilities have rendered most of the genome immune to introgression. 
These results support intrinsic postzygotic isolation as a driver of speciation in a moderately young cryptic species pair from the Amazon and suggest that species richness of the Amazon may be grossly underestimated.
                                        </div>
                                    </div>

                                </div>

                            </div>
                        </div>
                    </div>
                </div>
            </div>
            <hr>
        </div>




    </section>

    <!-- Footer -->
    <div id="footerJQ"></div>
    <script>
        $("#footerJQ").load("footer.html");
    </script>


    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
        integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
        crossorigin="anonymous"></script>
    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
        integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
        crossorigin="anonymous"></script>
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.4.1/dist/js/bootstrap.min.js"
        integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
        crossorigin="anonymous"></script>
</body>

</html>